app/views/taxon_ranges/_form_fields.html.haml

```diff
@@ -0,0 +1,33 @@
+.row.stacked
+  .col-xs-6
+    = f.text_field :taxon_id
+    - if f.object.range.file?
+      .field
+        = link_to f.object.range.url, f.object.range.url
+        = link_to_function t(:replace_parenthesis), "$(this).parents('.field').after('#{f.file_field(:range)}'); $(this).parents('.field').hide()", :class => "small"
+      = f.check_box :delete_range, :label_after => true
+    - else
+      = f.file_field :range
+    = f.form_field :iucn_relationship, :label => false do
+      = label_tag :is_this_an_iucn_redlist_range_map_and_if_not_why, t(:is_this_an_iucn_redlist_range_map_and_if_not_why)
+      .radio
+        %label.inline
+          = f.radio_button :iucn_relationship, 0, :skip_builder => true
+          = t(:this_is_an_iucn_redlist_range_map)
+      .radio
+        %label.inline
+          = f.radio_button :iucn_relationship, 1, :skip_builder => true
+          = t(:iucn_redlist_range_map_for_this_taxon_has_issues)
+      .radio
+        %label.inline
+          = f.radio_button :iucn_relationship, 2, :skip_builder => true
+          = t(:taxon_not_in_the_iucn_redlist)
```

Comment:
If you use `BootstrapFormBuilder` instead of `DefaultFormBuilder` you can re-write this as `= f.radio_button :iucn_relationship, 2, label: t(:taxon_not_in_the_iucn_redlist)`, plus you'll get the default styles for the other form fields. The source choose will cause problems, though. You might have to hard code the width of that input for it to work properly.